.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: You have run through the README.md for both Android and iOS and it still won't
+  work
+title: "\U0001F41E[Enter Bug Description]"
+labels: bug
+assignees: ''
+
+---
+
+- [x] I have read the [README](https://github.com/vrtdev/flutter_workmanager/blob/master/README.md)
+- [x] I have done the setup for [Android](https://github.com/vrtdev/flutter_workmanager/blob/master/ANDROID_SETUP.md)
+- [x] I have done the setup for [iOS](https://github.com/vrtdev/flutter_workmanager/blob/master/IOS_SETUP.md)
+- [x] I have ran the sample app and it does not work there
+
+**Version**
+| Technology | Version |
+| ------------------- | ------------- |
+| Workmanager version | |
+| XCode version       | |
+| Swift version       | |
+
+Test
```

Comment:
💁‍♂️

Reply:
Het render niet in Github for some reason

Reply:
Maybe look into ascii tables https://ozh.github.io/ascii-tables/